vtex/loaders/intelligentSearch/productListingPage.ts

```diff
@@ -203,7 +204,7 @@ const queryFromPathname = (
   const isPage = Boolean(pageTypes.length);
   const isValidPathSearch = pathList.length == 1;
   if (!isPage && !isInSeachFormat && isValidPathSearch) {
-    return pathList[0];
+    return decodeURI(pathList[0]);
```

Comment:
Can you clarify why we need this here?

A code comment would be nice

Reply:
Fixed